tools/ci_build/github/azure-pipelines/templates/win-ci.yml

```
@@ -32,6 +32,21 @@ jobs:
     CUDA_VERSION: ${{ parameters.CudaVersion }}
 
   steps:
+    - powershell: |
+        if($env:WINMLTELEMETRYGUID)
+        {
+          $length = $env:WINMLTELEMETRYGUID.length
+          $fileContent = "#define TraceLoggingOptionMicrosoftTelemetry() \
+            TraceLoggingOptionGroup("+$env:WINMLTELEMETRYGUID.substring(1, $length-2)+")"
+        }
+        else
+        {
+          $fileContent = "#define TraceLoggingOptionMicrosoftTelemetry() \
+            TraceLoggingOptionGroup(0000000000, 00000, 00000, 0000, 0000, 0000, 0000, 0000, 000, 0000, 0000)" 
+        }
```

no else.  we want this to "fail fast" since this is a bad config.